autoload/gutentags/ctags.vim

```diff
@@ -101,7 +101,7 @@ function! gutentags#ctags#generate(proj_dir, tags_file, write_mode) abort
             let l:cmd .= ' -o "' . l:proj_options_file . '"'
         endif
         for ign in split(&wildignore, ',')
-            let l:cmd .= ' -x ' . '"' . ign . '"'
+            let l:cmd .= ' -x ' . '"' . s:escape_patterns(ign) . '"'
```

Comment:
I wasn't aware of that functionality within Vim - I've tested it and it works as expected, with the latest changes pushed.